shub/config.py

```diff
@@ -81,11 +82,12 @@ def load(self, stream):
                 yaml_option_conf = yaml_cfg.get(option, {})
                 option_conf.update(yaml_option_conf)
                 if option == 'images' and yaml_option_conf:
-                    click.echo(
+                    warnings.warn(
```

Comment:
@vshlapakov @jdemaeyer what about memoizing the config upon first successful call? This way no extra args are needed, config is parsed only once.

I dislike how warnings look like when one calls the command compared to how it looked before.

```
→ shub deploy
/Users/alex/Dev/workspace/scrapinghub/repo/shub/shub/config.py:90: ShubDeprecationWarning: Images section is deprecated, please replace it with global `image` setting or define `image` setting for the project. Check for additional details in https://shub.readthedocs.io/en/stable/configuration.html.
  ShubDeprecationWarning
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
WARNING: Endpoint "devbox" is still using HTTP. Please change it to HTTPS if possible.
```